cleanlab/object_detection/rank.py

```diff
@@ -18,23 +18,27 @@
 are to contain label errors. """
 
 import warnings
+
 from cleanlab.internal.constants import (
     ALPHA,
-    LOW_PROBABILITY_THRESHOLD,
-    HIGH_PROBABILITY_THRESHOLD,
-    TEMPERATURE,
-    EUC_FACTOR,
-    MAX_ALLOWED_BOX_PRUNE,
-    CUSTOM_SCORE_WEIGHT_OVERLOOKED,
     CUSTOM_SCORE_WEIGHT_BADLOC,
+    CUSTOM_SCORE_WEIGHT_OVERLOOKED,
     CUSTOM_SCORE_WEIGHT_SWAP,
+    EUC_FACTOR,
+    HIGH_PROBABILITY_THRESHOLD,
+    LOW_PROBABILITY_THRESHOLD,
+    MAX_ALLOWED_BOX_PRUNE,
+    TEMPERATURE,
 )
 
-from cleanlab.internal.object_detection_utils import softmin1d
+global CUSTOM_SCORE_WEIGHT_OVERLOOKED, CUSTOM_SCORE_WEIGHT_BADLOC, CUSTOM_SCORE_WEIGHT_SWAP
```

Comment:
I would never do this, why did you need this instead of importing these values into the file where they are needed?

Reply:
setting anything global is quite dangerous

Reply:
Fixed in commit: https://github.com/cleanlab/cleanlab/pull/676/commits/289827ed8a5caf0c7dba4673cb63a894f75407a7